--- src/alpha.asm	2025-03-13 12:32:48
+++ alpha.asm	2025-03-13 12:33:23
@@ -1,16 +1,16 @@
 ; Set main
 section .text
-global  _main
+global  _start
 
 ; Main entrypoint
-_main:
+_start:
 	call print
 	call exit
 
 ; Print func
 print:
 	;   Sys write
-	mov rax, 0x02000004
+	mov rax, 1
 	;   To stdout
 	mov rdi, 1
 	;   Addr of alphabet
@@ -23,7 +23,7 @@
 ; Exit func
 exit:
 	;   Sys exit
-	mov rax, 0x02000001
+	mov rax, 60
 	;   Exit 0
 	xor rdi, rdi
 	syscall
